lib/src/anim_nav/animated_nav_tiles.dart

```diff
@@ -412,7 +415,7 @@ class _IconWidgetState extends State<IconWidget>
 
       animationColor = ColorTween(
         begin: widget.item.backgroundColor ?? widget.item.unSelectedColor,
-        end: widget.item.selectedColor,
+        end: widget.color,
```

Comment:
Replacing `item.selectedColor` with color might lead to unexpected behavior if the `color` parameter is not used correctly or consistently. It's unclear why this change was made without a clear explanation.